### PR TITLE
Remove protocol v1 and fix checkstyle

### PR DESCRIPTION
**Pull request by dmsolr**

### CODE REVIEW

**Comment by dmsolr** [comment]
I have tested in agent plugin.

**Comment by dmsolr** [comment]
```yaml
registryItems:
  services:
  - {canal-scenario: 2}
  instances:
  - {canal-scenario: 1}
  operationNames:
  - canal-scenario: [/canal-scenario/case/healthCheck, /canal-scenario/case/canal-case]
  heartbeat: []
segmentItems:
- serviceName: canal-scenario
  segmentSize: 2
  segments:
  - segmentId: 1.35.15833332977840000
    spans:
    - operationName: /canal-scenario/case/healthCheck
      operationId: 0
      parentSpanId: -1
      spanId: 0
      spanLayer: Http
      startTime: 1583333297791
      endTime: 1583333298137
      componentId: 1
      componentName: ''
      isError: false
      spanType: Entry
      peer: ''
      peerId: 0
      tags:
      - {key: url, value: 'http://localhost:8080/canal-scenario/case/healthCheck'}
      - {key: http.method, value: HEAD}
  - segmentId: 1.36.15833332982070000
    spans:
    - operationName: Canal/example
      operationId: 0
      parentSpanId: 0
      spanId: 1
      spanLayer: Unknown
      startTime: 1583333298302
      endTime: 1583333298326
      componentId: 54
      componentName: ''
      isError: false
      spanType: Exit
      peer: canal-server 172.21.0.3:11111
      peerId: 0
      tags:
      - {key: batchSize, value: '1000'}
      - {key: destination, value: example}
    - operationName: /canal-scenario/case/canal-case
      operationId: 0
      parentSpanId: -1
      spanId: 0
      spanLayer: Http
      startTime: 1583333298207
      endTime: 1583333298382
      componentId: 1
      componentName: ''
      isError: false
      spanType: Entry
      peer: ''
      peerId: 0
      tags:
      - {key: url, value: 'http://localhost:8080/canal-scenario/case/canal-case'}
      - {key: http.method, value: GET}
```

File: validator/src/main/java/org/apache/skywalking/plugin/test/agent/tool/validator/assertor/ApplicationAssert.java [hunk 1]
```diff
@@ -42,7 +42,7 @@ public static void assertEquals(List<RegistryApplication> expected, List<Registr
     }
 
     private static RegistryApplication getMatchApplication(List<RegistryApplication> actual,
-        RegistryApplication application) {
+                                                           RegistryApplication application) {
```
Comment (wu-sheng) [inline validator/src/main/java/org/apache/skywalking/plugin/test/agent/tool/validator/assertor/ApplicationAssert.java]: @dmsolr I mean such as this file name. Application* is the old name, right?

File: validator/src/main/java/org/apache/skywalking/plugin/test/agent/tool/validator/assertor/exception/RegistryOperationNamesOfApplicationNotFoundException.java [hunk 2]
```diff
@@ -29,7 +29,8 @@ public RegistryOperationNamesOfApplicationNotFoundException(RegistryOperationNam
 
     @Override
     public String getCauseMessage() {
-        return String.format("RegistryOperationNamesOfApplicationNotFoundException %s\nexpected: %s\nactual: %s\n", applicationCode
-            .applicationCode(), applicationCode.operationName(), "NOT FOUND");
+        return String.format(
+            "RegistryOperationNamesOfApplicationNotFoundException %s\nexpected: %s\nactual: %s\n", applicationCode
```
Comment (wu-sheng) [inline validator/src/main/java/org/apache/skywalking/plugin/test/agent/tool/validator/assertor/exception/RegistryOperationNamesOfApplicationNotFoundException.java]: You miss this.

File: validator/src/main/java/org/apache/skywalking/plugin/test/agent/tool/validator/assertor/exception/RegistryInstanceOfApplicationNotFoundException.java [hunk 3]
```diff
@@ -29,6 +29,9 @@ public RegistryInstanceOfApplicationNotFoundException(String applicationCode) {
 
     @Override
     public String getCauseMessage() {
-        return String.format("RegistryInstanceOfApplicationNotFoundException\nexpected: Instances of Service(%s)" + "\nactual: %s\n", applicationCode, "NOT FOUND");
+        return String.format(
+            "RegistryInstanceOfApplicationNotFoundException\nexpected: Instances of Service(%s)" + "\nactual: %s\n",
```
Comment (wu-sheng) [inline validator/src/main/java/org/apache/skywalking/plugin/test/agent/tool/validator/assertor/exception/RegistryInstanceOfApplicationNotFoundException.java]: And this

File: mock-collector/src/test/resources/expect-data.yaml [hunk 4]
```diff
@@ -15,26 +15,26 @@
 # limitations under the License.
 expectData:
   - registryItem:
-    - applications:
-      - test-applicationCode1: nq 0   // the application id of test-applicationCode is 3
-      - test-applicationCode2: nq 0
+    - services:
+      - test-serviceName1: nq 0   // the application id of test-serviceName is 3
```
Comment (wu-sheng) [inline mock-collector/src/test/resources/expect-data.yaml]: You miss this.

File: mock-collector/src/test/resources/sample-data.yaml [hunk 5]
```diff
@@ -16,26 +16,26 @@
 #
 validateData:
   - registryItem:
-    - applications:
-      - test-applicationCode1: 3   // the application id of test-applicationCode is 3
-      - test-applicationCode2: 1
+    - services:
+      - test-serviceName1: 3   // the application id of test-serviceName is 3
```
Comment (wu-sheng) [inline mock-collector/src/test/resources/sample-data.yaml]: This

**Comment by wu-sheng** [comment]
Merged, please forward to main repo and lua repo.

**Comment by wu-sheng** [comment]
Please use `commit id download` rather than `git submodule` in the main repo please.

**Comment by dmsolr** [comment]
> Please use `commit id download` rather than `git submodule` in the main repo please.

Got it.

**Pull request merged**